tests/test_cdg.py

```diff
@@ -17,9 +17,7 @@ def test_graph_0():
     # etc.
 
     # Create a project with a random binary - it will not be used anyways
-    p = angr.Project(os.path.join(test_location, 'x86_64', 'datadep_test'),
-                     load_options={'auto_load_libs': False},
-                     use_sim_procedures=True)
+    p = angr.Project(os.path.join(test_location, 'x86_64', 'datadep_test'),load_options={'auto_load_libs': False},use_sim_procedures=True)
```

Comment:
Please avoid unnecessary reformatting

Reply:
I have fixed the reformatting issue. please let me know if something is missing.

Reply:
This isn't fixed. The original formatting for this function was fine, so simply reverting the changes to as they were is ideal.